src/index.html

```diff
@@ -2,13 +2,33 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="./images/logo.png" 
+          class="logo" 
+          alt="moyo"
+        >
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="links"><a href="#apple" class="link_a is-active">APPLE</a></li>
+          <li class="links"><a href="#samsung" class="link_a">SAMSUNG</a></li>
+          <li class="links"><a href="#smartphones" class="link_a">SMARTPHONES</a></li>
+          <li class="links" data-qa="hover"><a href="#laptops" class="link_a">LAPTOPS & COMPUTERS</a></li>
+          <li class="links"><a href="#gadgets" class="link_a">GADGETS</a></li>
+          <li class="links"><a href="#tablets" class="link_a">TABLETS</a></li>
+          <li class="links"><a href="#photo" class="link_a">PHOTO</a></li>
+          <li class="links"><a href="#video" class="link_a">VIDEO</a></li>
```

Comment:
You've set `uppercase` - that's the correct way to do it but in this case, no need to do it in HTML
```suggestion
          <li class="links"><a href="#video" class="link_a">Video</a></li>
```